spec/request/gi/institutions_request_spec.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+# frozen_string_literal: false
```

Comment:
Can't do `force_encoding` on a frozen string literal, which is needed to test this.